packages/react-storage/src/components/StorageBrowser/Views/Controls/Table.tsx

```diff
@@ -280,6 +414,14 @@ export const LocationDetailViewTable = (): JSX.Element => {
   const hasItems = !!data.result?.length;
   const shouldReset = !history.length && hasItems && !location;
 
+  const [tableData, setTableData] = React.useState<LocationItem[]>(
+    sortFn(data.result, direction, selection)
+  );
+
+  React.useEffect(() => {
+    setTableData(sortFn(data.result, direction, selection));
+  }, [data.result, direction, selection, sortFn]);
```

Comment:
I thought I needed to do that to make the table re-render

I removed this these lines though and tried it out with just `const tableData = sortFn(data.result, direction, selection);` and can see that the table still re-renders! I'm guessing it's because my sort state gets updated and that should be enough to cause it render again?


 Sorry, just a misunderstanding on my part with regards to how to get it to re-render